frontend/src/pages/Graphing/components/Graph.tsx

```diff
@@ -166,6 +166,16 @@ const durationUnitMap: [number, string][] = [
 	[24, 'd'],
 ]
 
+const timeMetrics = {
```

Comment:
:+1: nice, was going to do something similar for session length / active length

Reply:
@mayberryzane any extra keys i should add here in this pr?